darts/models/__init__.py

```diff
@@ -3,15 +3,37 @@
 ------
 """
 
+from ..logging import get_logger
+
+logger = get_logger(__name__)
+
 # Forecasting
-from .arima import ARIMA, AutoARIMA
+try:
+    from arima import AutoARIMA
+except ModuleNotFoundError:
+    logger.warning("Support pmdarima based models not available. To enable it install darts[pmdarima]")
+
+try:
+    from .prophet import Prophet
+except ModuleNotFoundError:
+    logger.warning("Support Prophet based models not available. To enable it install darts[prophet]")
+
+try:
+    from .rnn_model import RNNModel
+except ModuleNotFoundError:
+    logger.warning("Support Torch based models not available. To enable it install darts[torch]")
+
+try:
+    from .tcn_model import TCNModel
+except ModuleNotFoundError:
+    logger.warning("Support Torch based models not available. To enable it install darts[torch]")
```

Comment:
@pennfranc I placed all the extras in `try` blocks so that the import interface is kept.  
I did something similar by moving `AutoARIMA` outside `arima.py`. This should have the effect that if the correct deps are installed then `from arima import *` should import the same forecaster classes.

Reply:
@plorenzatto Sorry for getting back to you so late!
Separating `AutoARIMA` and `ARIMA` seems like a good move to me given the different dependencies. The `try` block structure also seems like a good idea as far as I can see (thoughts @hrzn?).